Documentation/heisenbug.rst

```diff
@@ -10,19 +10,26 @@ It also has 2 GPGPUs (NVIDIA GeForce RTX 3090), that can be used to run the GPU
 The RTX 3090 belongs to a consumer kind of graphics cards and thus does not perform well with double precision. 
 Therefore, it is preferable to compile SeisSol with single precision.
 
-A module integrating all libraries relevant for SeisSol-GPU is preinstalled at ``/export/dump/ravil/modulefiles``.
+A module integrating all libraries relevant for compiling SeisSol with CUDA and SYCL is available on heisenbug.
 It can be discovered at startup after adding the following to ``~/.bashrc``:
 
 .. code-block:: bash
 
-    module_hpcsdk=/export/dump/ravil/modulefiles
-    export MODULEPATH=$MODULEPATH:${module_hpcsdk}
+    module use /import/exception-dump/ulrich/spack/modules/linux-debian11-zen2
 
 It is then loaded with:
 
 .. code-block:: bash
 
-    module load seissol-env-gcc-11.1.0
+    # load the (first in the list) seissol-env module compiled with cuda support
+    module load $(module avail seissol-env/*-cuda-* | awk '/seissol-env/ {print $1}')
```

Comment:
Are you sure you want the first in list? (and not the last in list with such?)

Reply:
Well anyway, there should be not several of them. 
I just don't want to give a hash number, mess with my spack setup, reinstall seissol-env, and have the wrong hash in the documentation.

But it i seems the first one is loaded. See the following test:
```
(base) ulrich@heisenbug:~$ module avail seissol-env/*
------------------------------------------------------------------------ /import/exception-dump/ulrich/spack/modules/linux-debian11-zen2 ------------------------------------------------------------------------
seissol-env/develop-gcc-10.2.1-cuda-qb6oglb  seissol-env/develop-gcc-12.2.0-f4bnvcu  
(base) ulrich@heisenbug:~$ module avail seissol-env/* | awk '/seissol-env/ {print $1}'
seissol-env/develop-gcc-10.2.1-cuda-qb6oglb
```